opflexagent/vif_plug_vpp/linux_net.py

```diff
@@ -42,7 +41,7 @@ def _create_vpp_vif(dev, iface_id, mac, mtu,
                         1, mac_address, iface_id)
     LOG.debug("sw_if_index:{}".format(sw_if_index))
     if mtu:
-        vapi.set_interface_mtu(sw_if_index, mtu)
+        vapi.set_interface_mtu(sw_if_index, [mtu, 0, 0, 0])
```

Comment:
not sure what the extra 3 0's mean here.....

Reply:
4 values are for [all packets, L3 packets, IPv4 packets, MPLS packets], first value applies to all unless overriden.